recipe/meta.yaml

```diff
@@ -45,35 +44,37 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=10.0.1,<11
-    - pyarrow
-    - numpy
-    - setuptools >=40.6.0
+    - arrow-cpp 10.0.1
+    - pyarrow  10.0.1
+    - numpy  {{ numpy }}
+    - setuptools
     - wheel
   run:
     - python
     - packaging
-    - pyarrow
+    - pyarrow  >=10.0.1,<10.1.0
```

Comment:
`pyarrow` is actually only needed as an [optional requirement](https://github.com/snowflakedb/snowflake-connector-python/blob/1380e41cd2088f0c83af0cf45010d68aac96b366/setup.cfg#L98).

Reply:
Yes, thanks!